public/js/encoder.js

```diff
@@ -76,7 +75,7 @@ async function encoder(vc, VideoClient) {
   let broadcasting = false;
   // Click handler for creating the call and broadcasting/ending the broadcast/closing the call.
```

Comment:
I would add commenting like this in the other files to explain what you are doing, although it is a bit over the top I think it is really useful for learning.